spec/models/match_spec.rb

```diff
@@ -43,6 +43,23 @@
     end
   end
 
+  context '#winner' do
+    before do
+      @undecided_match = create(:undecided_group_match)
```

Comment:
1

Reply:
Well okay I apparently can't read, i looked at the other context.. sorry; gonna change it

Reply:
done